### PR TITLE
Adds ICC, a new hostile ERT, and a minor SoM blurb change.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds ICC (Independent Colonial Confederation) which is a new Hostile ERT, their allies are the SoM in alignment, they're hostile to about everyone else.
Adds ICC jobs which are the standards, leaders, guards and medics.
Adds 2 ICC guns. The Trenchgun for the leader, and the coilgun for the Guardsman.

ICC have bring an incredibly high amount of explosives with them, with high bomb resist to help them create mayhem on a HvH ship activity. They also all have 2 engineering and 2 construction skill outside of the medics, who have much lower.
ICC

Spriter Credits
-
Armor sprites/Uniform/Bag by Fitz.
Trenchgun by Crow
Coilgun stolen from the kuro gun PR that ended up reverted. That's a placeholder for now.
Belt/Pouches by me if original, from kuro if a recolor.

## Why It's Good For The Game
New ERT which is the actual main enemy of TGMC in the lore. So that's cool. Also I w
## Changelog
:cl:
add: Added new Hostile ERT, ICC. They're allied to SoM and are strapped with a large amount of explosives and engineering equipment.
spellcheck: SoM ert blurb has been redone to clairify a few things, such as the addition of ICC as an ally and grammar stuff.
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]

File: code/modules/projectiles/guns/specialist.dm [hunk 1]
```diff
@@ -857,3 +857,42 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
 	recoil = 0
 	scatter = 0
 	movement_acc_penalty_mult = 6
+
+//-------------------------------------------------------
+//ML-120 Coilgun
+
+/obj/item/weapon/gun/rifle/icc_coilgun
+	name = "\improper ML-120 coilgun"
+	desc = "The ML-120 coilgun is the most common seen coilgun in ICCAF use, firing magnetic projecitles at a incredibly high velocity, requires some windup but will penetrate walls, your foes, and your friendlies too. So watch out.. Uses specialized canisters to reload."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/specialist.dm:866]:
```suggestion
	desc = "The ML-120 coilgun is the most commonly seen coilgun in ICCAF use, firing magnetic projecitles at a incredibly high velocity. It requires some windup but will penetrate walls, your foes, and your friendlies too. So watch out... Uses specialized canisters to reload."
```

**Comment by FosterGrime-Heymynameslouie** [comment]
should replace clf tbh
very cool

**Comment by zzurge** [comment]
As much as i love this its a criminal shame that i may never see them in a match

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -717,3 +717,18 @@
 	icon_state = "robot_helmet_heavy"
 	item_state = "robot_helmet_heavy"
 
+// ICC. Their helmets have higher bomb resist.//
+
+/obj/item/clothing/head/helmet/marine/icc
```
[the remark this answers is not in the document]
Comment (Pariah919) [reply code/modules/clothing/head/helmet.dm:722]: I'll normalize it with the armor.

File: code/modules/clothing/suits/marine_armor.dm [hunk 3]
```diff
@@ -885,6 +885,23 @@
...
+	flags_cold_protection = CHEST|GROIN|ARMS|LEGS|FEET|HANDS
+	flags_heat_protection =CHEST|GROIN|ARMS|LEGS|FEET|HANDS
+	soft_armor = list(MELEE = 50, BULLET = 60, LASER = 50, ENERGY = 60, BOMB = 70, BIO = 10, FIRE = 60, ACID = 50)
+	flags_item_map_variant = NONE
+
+/obj/item/clothing/suit/storage/marine/icc/guard
+	name = "\improper Modelle/19 combat armor"
+	desc = "A piece of ICCGF body armor, worn by specialized infantry. Most Infantry actions in the ICC forces are done by adhoc personnel due to constant shortages of manpower, however most real Infantry divisions are of high quality, and are better known as 'Guardsmen'.  Protects well from most sources, and will entirely protect from explosions."
+	icon_state = "icc_guard"
+	soft_armor = list(MELEE = 60, BULLET = 65, LASER = 40, ENERGY = 60, BOMB = 100, BIO = 10, FIRE = 55, ACID = 40)
```
[the remark this answers is not in the document]
Comment (Pariah919) [reply code/modules/clothing/suits/marine_armor.dm:903]: I'll drop it to 85, you're probably right.

**Comment by FIXTHERADBOXPLEASE** [comment]
Dripmongers deployed.

File: code/modules/projectiles/ammo_datums.dm [hunk 4]
```diff
@@ -1670,6 +1670,24 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 		return
 	T.ignite(5, 10)
 
+
+/datum/ammo/bullet/coilgun
```
[the remark this answers is not in the document]
Comment (Pariah919) [reply code/modules/projectiles/ammo_datums.dm:1674]: Removed wallpierce.

**Comment by Al-1ce** [comment]
It would be wise to add a small blurb to all SoM that the ICC are their allies.

**Comment by ThelastArchiver** [comment]
the return of the knights

File: code/datums/jobs/job/icc.dm [hunk 5]
```diff
@@ -0,0 +1,221 @@
+/datum/job/icc
+	job_category = JOB_CAT_MARINE
+	access = ALL_ANTAGONIST_ACCESS
+	minimal_access = ALL_ANTAGONIST_ACCESS
+	skills_type = /datum/skills/crafter
+	faction = FACTION_USL
```
Comment (Lumipharon) [inline code/datums/jobs/job/icc.dm:6]: bruh

File: code/datums/jobs/job/icc.dm [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+	outfit = /datum/outfit/job/icc/standard
+
+/datum/outfit/job/icc
+	name = "ICC Standard"
+	jobtype = /datum/job/icc
+
+	id = /obj/item/card/id/silver
+	w_uniform = /obj/item/clothing/under/icc/webbing
+	belt = /obj/item/storage/belt/marine/icc
+	ears = /obj/item/radio/headset/distress/som
```
Comment (Lumipharon) [inline code/datums/jobs/job/icc.dm:21]: wuh

File: code/datums/jobs/job/icc.dm [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+	H.equip_to_slot_or_del(new /obj/item/storage/box/m94, SLOT_IN_BACKPACK)
+
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som, SLOT_IN_R_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som, SLOT_IN_R_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som, SLOT_IN_R_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som, SLOT_IN_R_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som/thermobaric, SLOT_IN_L_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som/thermobaric, SLOT_IN_L_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som/heat, SLOT_IN_L_POUCH)
+	H.equip_to_slot_or_del(new /obj/item/ammo_magazine/rocket/som/heat, SLOT_IN_L_POUCH)
```
Comment (Lumipharon) [inline code/datums/jobs/job/icc.dm:113]: 25% of the entire ert all having HE and thermobaric rockets is an extremely bad idea.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Review by Lumipharon — APPROVED** [review]
RPG chance is honestly still kinda high, you should really just add more loadouts. Should be fine for testing though.

**Pull request merged**